src/components/Counter.component.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class Counter extends Component {
+  state = { count: this.props.count }
+
+  handleIncrementClick = () => {
+    const { count } = this.state
+    const currentCount = count
+    this.setState({ count: 1 + currentCount })
+  }
+
+  handleDecrementClick = () => {
+    const { count } = this.state
+    const currentCount = count
+    this.setState({ count: currentCount - 1 })
+  }
+
+  render() {
+    const { count } = this.state
+    return (
+      <div>
+        <button onClick={this.handleIncrementClick}> Increment </button>
+        <label>{count}</label>
```

Comment:
The `label` element is usually used in conjunction with `input` elements. Use a `div` here instead.